avocado/core/suite.py

```diff
@@ -22,6 +22,7 @@
 from .loader import (DiscoverMode, LoaderError, LoaderUnhandledReferenceError,
                      loader)
 from .parser import HintParser
+from .requirements.resolver import create_requirements_tasks
 from .resolver import resolve
```

Comment:
We have a module named `resolver` and now another one named `requirements.resolver`. Hum.. just thinking out of loud. What about having a package `resolvers.` with all the resolvers there? 

Yes, maybe too much work for now. Just an idea.

Reply:
I agree! I initially thought these 2 would be the only resolvers, but who knows :)

Reply:
Are they doing the same things though? I am not as accustomed to the code as the two of you but AFAIR `resolver` was used just as a new word for test `loader` to differentiate from the old runner and the current resolver is a specialized class for fulfilling test requirements.

Reply:
Well, if we say that a resolver is "resolving" something from one representation to another, I would agree that our resolvers are doing the same.

Reply:
Right but I prefer not to attach to the word itself especially considering how ubiquitous its use has become thus making it lose a lot of its technical meaning to begin with. A far better approach IMO would be to later rename the "resolver" in the other meaning back to "loader" and restore the original loader/runner pairing. In this way both modules meanings are differentiated as they should be instead of going the opposite way and even putting them in the same subpackage as if they are variations or components of the same functional goal.

Reply:
But I might misunderstand, both way would work and worse case one would create the need of more refactoring much later on once their roles are better established.